### ISSUE TITLE
Validation / Training Dataset / Dataloader for Speech VAD Models

### ISSUE THREAD

**Issue by crutcher** [zspacelabs/bunsen#125]
`bunsen::kits::speech::silero_vad` is already loadable.
`ten_vad` is in-flight.

validating performance (accuracy / speed) under different tuning choices needs a dataset.
training and cross-comparisons also need a dataset.

I'm looking at the LibreVAD dataset; but I need a batch builder for bunsen/burn.
